app/src/ui/history/commit-list.tsx

```diff
@@ -350,19 +354,42 @@ export class CommitList extends React.Component<ICommitListProps, {}> {
     return this.props.commitSHAs.findIndex(s => s === sha)
   }
 
+  private getRowCustomClassMap = () => {
+    const { commitSHAs, shasToHighlight } = this.props
+    if (shasToHighlight.length === 0) {
+      return undefined
+    }
+
+    const rowsForShasNotInDiff = commitSHAs
+      .filter(sha => shasToHighlight.includes(sha))
+      .map(sha => this.rowForSHA(sha))
+
+    if (rowsForShasNotInDiff.length === 0) {
+      return undefined
+    }
+
+    const rowClassMap = new Map<string, ReadonlyArray<number>>()
+    rowClassMap.set('highlighted', rowsForShasNotInDiff)
+    return rowClassMap
+  }
+
   public render() {
-    if (this.props.commitSHAs.length === 0) {
-      return (
-        <div className="panel blankslate">{this.props.emptyListMessage}</div>
-      )
+    const { commitSHAs, selectedSHAs, shasToHighlight, emptyListMessage } =
+      this.props
+    if (commitSHAs.length === 0) {
+      return <div className="panel blankslate">{emptyListMessage}</div>
     }
 
+    const classes = classNames({
+      'has-highlighted-commits': shasToHighlight.length > 1,
```

Comment:
I ran into a case where only 1 commit was reachable, and this was preventing it from being highlighted:
```suggestion
      'has-highlighted-commits': shasToHighlight.length > 0,
```